app/logic/fileHandler.py

```diff
@@ -12,41 +12,54 @@ def __init__(self,files=None, courseId=None, eventId=None):
         if courseId:
             self.path = os.path.join(self.path, app.config['files']['course_attachment_path'], str(courseId))
         elif eventId:
-            self.path = os.path.join(self.path, app.config['files']['event_attachment_path'], str(eventId))
+            self.path = os.path.join(self.path, app.config['files']['event_attachment_path'])       #Removed event ID from how the file path is being saved.
+        
         try:
-            os.makedirs(self.path)
+            extraDir = str(self.eventId) if self.eventId else ""
+            os.makedirs(os.path.join(self.path, extraDir))                                          #Changed the path where the directory is created.
```

Comment:
That comment isn't needed. It doesn't really add any information other than that you have made changes. I would either get rid of it or make it "Create the directory for the event or course". And don't put it so far at the end of the line. Lines shouldn't be so long that they wrap. If it doesn't fit, put it before the code it is commenting.